docs/ui/insight/tools/connection-management.md

```diff
@@ -0,0 +1,14 @@
+
+---
+title: Connection management
+linkTitle: Connections
+description: Manage connections in RedisInsight
+weight: 10
+
+---
+
+* Automatically discover and add your local Redis or Redis Stack databases (that use standalone connection type and do not require authentication).
+* Discover your databases in Redis Enterprise Cluster and databases with Flexible plans in Redis Cloud.
+* Use a form to enter your connection details and add any Redis database running anywhere (including OSS Cluster, Sentinel).
+
```

Comment:
RedisInsight allows users to connect to multiple Redis and Redis Stack instances and manage them from a single interface.
While all your local Redis or Redis Stack databases with the standalone connection type and without any authentication will be automatically identified and added to your list of database connections when RedisInsight launches for the first time, there are many cases when you need to add database connections to work with them via RedisInsight.
To do so, you can use one of the available options:
* Discover your databases in Redis Enterprise Cluster and databases with Flexible plans in Redis Cloud, as well as primary groups from Redis Sentinel
* Use a form to enter your connection details and add any Redis database running anywhere

RedisInsight also supports database connections via SSH tunnels using a password or private key in PEM format.